lib/reversetunnel/agent.go

```diff
@@ -361,6 +357,7 @@ func (a *Agent) startHeartbeat(conn ssh.Conn) {
 			errC <- err
 			return
 		}
+		hb.SendRequest("ping", false, nil)
```

Comment:
see the line above - there's already such request executed, and your line adds one more